To: vim_dev@googlegroups.com
Subject: Patch 7.3.383
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.383
Problem:    For EBCDIC pound sign is defined as 't'.
Solution:   Correctly define POUND.
Files:	    src/ascii.h


*** ../vim-7.3.382/src/ascii.h	2011-09-02 14:18:14.000000000 +0200
--- src/ascii.h	2011-12-17 21:38:36.000000000 +0100
***************
*** 123,129 ****
  #define DCS		0x90	/* Device Control String */
  #define STERM		0x9c	/* String Terminator */
  
! #define POUND		'\xA3'
  
  #define CTRL_F_STR	"\056"
  #define CTRL_H_STR	"\026"
--- 123,129 ----
  #define DCS		0x90	/* Device Control String */
  #define STERM		0x9c	/* String Terminator */
  
! #define POUND		0xB1
  
  #define CTRL_F_STR	"\056"
  #define CTRL_H_STR	"\026"
*** ../vim-7.3.382/src/version.c	2011-12-15 21:51:32.000000000 +0100
--- src/version.c	2011-12-23 12:46:08.000000000 +0100
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     383,
  /**/

-- 
Vim is like Emacs without all the typing.  (John "Johann" Spetz)

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
